releases/HEAD/release-ilcsoft.cfg

```diff
@@ -241,6 +241,16 @@ ilcsoft.module("DD4hep").envcmake["DD4HEP_USE_GEAR"]=1
 
 ilcsoft.install( lcgeo( lcgeo_version )) 
 
+
+ilcsoft.install( podio( podio_version ))
+ilcsoft.module("podio").envcmake["ENABLE_SIO"] = 'ON'
+ilcsoft.module("podio").envcmake["USE_EXTERNAL_CATCH2"] = 'OFF'
+
+ilcsoft.install( edm4hep( edm4hep_version ))
+ilcsoft.module("edm4hep").envcmake["BUILD_DDG4EDM4HEP"] = 'OFF'
```

Comment:
Not that it matters for this PR, more out of curiosity: Does EDM4hep still have this option? (I hope not)